### ISSUE TITLE
Issue with Line 184 in transaction_builder ?

### ISSUE THREAD

**Issue by brianoflondon** [emre/lighthive#13]
Since this morning I'm getting this in the runtime output of some of my code:

```
/usr/local/lib/python3.11/site-packages/lighthive/broadcast/transaction_builder.py:184: SyntaxWarning: "is" with a literal. Did you mean "=="?
```

Code seems to be running fine. I'm using Python 3.11. I'm not sure why I didn't see this before though it only occurs when my FastAPI code is starting up.

**Comment by brianoflondon** [comment]
Still seeing this all over the place when using Lighthive. Doesn't cause any issues but looks bad in logs.

**Comment by emre** [comment]
Thanks for reporting. Having a look.

**Comment by emre** [comment]
addressed w// v0.4.3.
